interface/interface.yaml

```diff
@@ -0,0 +1,61 @@
+components:
+  schemas:
+    Interface:
+      description: >-
+        Local Interface for Testbed Reservation Service.
+      type: object
+      properties:
+        name:

```

Comment:
Similar to using name and id for devices, we shall adopt similar approach for interface names in the inventory, and logical interface IDs.

Reply:
changed `name` to `id`

Reply:
Similar to device names, we need to provide an option for users to specify interface `name` for the search. This is even more important as for devices, as it allows to select interfaces with specific affinity to a pipeline, ASIC etc. So, use `id` as a unique logical identifier, and `name` to search.

Reply:
Added `name` attribute